docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -25,6 +25,13 @@ cluster in a single JSON file.
 Configuration
 -------------
 
+.. warning::
+
+    From Trino 401 the default access to PTFs (Polymorphic Table Function) has
```

Comment:
```suggestion
    As of Trino 401, the default access to PTFs (Polymorphic Table Function) has
```

Reply:
I mix this with https://github.com/trinodb/trino/pull/14774#discussion_r1009694755